xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-picker/xwiki-platform-attachment-picker-ui/src/main/resources/Attachment/Picker/Code/CKEditorImagePlugin.xml

```diff
@@ -134,6 +134,33 @@
     picker.on('xwiki:attachmentGalleryPicker:unselected', function() {
       imageSelector.updateSelectedImageReferences([], $(this));
     });
+    // We extend the attachment picker to allow quick form submission
+    const results = picker.find('.attachmentPickerResults');
+    let selectButton = element.parents('.modal-content').find('.modal-footer .btn-primary');
+    function validateForm(event) {
+      event.preventDefault();
+      // Make sure we have a selected picture before going forward
+      let selected = $(event.currentTarget).parents('.attachmentGroup').data('id');
+      imageSelector.updateSelectedImageReferences([selected], picker);
+      // Validate the selection of the double click picture.
```

Comment:
This comment is not correct, this code is also used for the enter key.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2817/commits/9f172766d215ed68b89ceb9bdbe883a4cc50c599 :+1: